Sources_v3/Workers/EventObservers/EventObserver_Tests.swift

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient
+import XCTest
+
+final class EventObserver_Tests: XCTestCase {
+    var notificationCenter: NotificationCenter!
+    var eventToDeliver: HealthCheckEvent!
+    var observer: EventObserver?
+    var eventNotification: Notification {
+        .init(newEventReceived: eventToDeliver, sender: self)
+    }
+
+    // MARK: - Setup
+
+    override func setUp() {
+        super.setUp()
+
+        notificationCenter = NotificationCenter()
+        eventToDeliver = HealthCheckEvent(connectionId: .unique)
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
nit: `super.tearDown()` should be called at the end of this function. You want to clean up your things first before the superclass cleans its things. In this very case, it doesn't really matter, since it's not mandatory to call super when subclassing from `XCTestCase`. However, sometimes we use a different superclass (like `StressTestCase`) and it can matter there. It's just a best practice we established internally to save us from headaches in the future.